configs/jest.js

```diff
@@ -0,0 +1,20 @@
+import jestPlugin from 'eslint-plugin-jest';
+
+export default [
+  {
+    files: ['test/**', '__mocks__/**', '**/*.spec.ts', '**/*.test.ts'],
```

Comment:
```suggestion
    files: ['test/**', '__mocks__/**', '**/*.spec.{ts,tsx}', '**/*.test.{ts,tsx}'],
```

to support JSX in React codebases out of the box

Reply:
Done here: https://github.com/ridedott/eslint-config/pull/2567/commits/ff485efc6d0a3f38ef258bf5ea6e859ace1685ad